tests/Commands/DrupalCommandsTest.php

```diff
@@ -68,11 +68,61 @@ public function testPermissions(array $config, $command, $expected_error, $expec
         $this->assertEquals($expected_settings_file_permission, $siteSettingsPermissions);
     }
 
+    /**
+     * Test the services file setup.
+     *
+     * @param array $config
+     * @param array $data
+     * @param array $expected
+     *
+     * @dataProvider servicesSetupDataProvider
+     */
+    public function testServicesSetup(array $config, array $data, array $expected)
+    {
+        $configFile = $this->getSandboxFilepath('runner.yml');
+        file_put_contents($configFile, Yaml::dump($config));
+
+        // Prepare data for assertions depending if service filename is given in
+        // command or in runner Yaml file.
+        if (!empty($data['service-parameter'])) {
+            $services_source_file = $this->getSandboxFilepath($data['service-parameter']);
+            $service_parameter = ' --service-parameters=' . $data['service-parameter'];
+        } else {
+            $services_source_file = $this->getSandboxFilepath($config['drupal']['service_parameters']);
+            $service_parameter = '';
```

Comment:
We should initialize this variable outside of the `if`

Reply:
In this case I prefer like this since there is a parallelism between the if and the else blocks.

Reply:
It's your code, fair enough.